github-actions-scripts/folioci_npmver.sh

```diff
@@ -27,8 +27,10 @@ fi
 
 new_cur_ver=${maj_ver}.${min_ver}.${patch_ver}
 
-# add 000+Jenkins JOB_ID to current patch version
+# add 00009999+CI JOB_ID to current patch version
+# 9999 is here due to a change in CI workflows (STRIPES-904) which reset job IDs;
+# without them, newer builts may have smaller version numbers
 
-new_snap_ver=${new_cur_ver}00000${JOB_ID}
+new_snap_ver=${new_cur_ver}00009999${JOB_ID}
```

Comment:
@ncovercash I'd like to continue using some `0`s as a separator before the build number, even if it's gross and continues to highlight the ugliness here. It makes it easier to correlate the GA runs with what gets published in nexus and I'd like to preserve that. If we use `0900000` then we'll get values like this: 

```
1000001234     <<< current number we must exceed
1090000012
1090000099
10900000100
10900000101
```
compared to values like this where the 9s potentially run together:
```
1000001234     <<< current number we must exceed
10000999912
10000999999
100009999100
100009999101
```
Or should we use `0100000` so we have 8 more opportunities to do this wrong before actually implementing a proper `-beta` or `-snapshot` semver scheme? 😆